examples/miscellaneous/plot_kernel_approximation.py

```diff
@@ -71,19 +71,25 @@
 # data_test = scaler.transform(data_test)
 
 # Create a classifier: a support vector classifier
-kernel_svm = svm.SVC(gamma=0.2)
-linear_svm = svm.LinearSVC(dual="auto")
+kernel_svm = svm.SVC(gamma=0.2, random_state=42)
```

Comment:
We don't need it here. Only used when `probability=True`. Since this is the default, we don't use them.

```suggestion
kernel_svm = svm.SVC(gamma=0.2)
```